build/bundle.Dockerfile

```diff
@@ -29,6 +29,7 @@ LABEL name=gatekeeper/gatekeeper-operator-bundle
 LABEL description="The Gatekeeper Operator installs and configures Open Policy Agent Gatekeeper."
 LABEL summary="Red Hat Gatekeeper Operator"
 LABEL version=v3.17.1
+LABEL release="0"
```

Comment:
Just out of curiousity, what does this number track? Does it get automatically updated somewhere?

Reply:
I've been looking into it. Currently it's just a meaningless requirement from EC. Apparently there have been discussions around it but nothing concrete, for example:
- https://redhat-internal.slack.com/archives/C04PZ7H0VA8/p1725462167035529